test/avi/t_insert_mode.clj

```diff
@@ -130,6 +130,8 @@
                     "~" :blue
                     "test.txt" :black :on :white
                     ""])
+  (fact "repeat count for `i` with `3ix<Esc>` leaves the point on last `x` added"
```

Comment:
If you want to add a test but aren't ready to implement it, you can replace the word `fact` with the word `future-fact`, and it won't fail the build.  It will note there's something left to do.